README.md

```diff
@@ -255,552 +197,41 @@ apollo {
-
-```groovy
-dependencies {
-  implementation 'com.apollographql.apollo:apollo-http-cache:x.y.z'
-}
-```
-
-Raw HTTP Response Cache:
-
-```java
-//Directory where cached responses will be stored
-File file = new File(context.getApplicationContext().getFilesDir(), "apolloCache");
-
-//Size in bytes of the cache
-long size = 1024*1024;
-
-//Create the http response cache store
-DiskLruHttpCacheStore cacheStore = new DiskLruHttpCacheStore(file, size); 
-
-//Build the Apollo Client
-ApolloClient apolloClient = ApolloClient.builder()
-  .serverUrl("/")
-  .httpCache(new ApolloHttpCache(cacheStore))
-  .okHttpClient(okHttpClient)
-  .build();
-
-apolloClient
-  .query(
-    FeedQuery.builder()
-      .limit(10)
-      .type(FeedType.HOT)
-      .build()
-  )
-  .httpCachePolicy(HttpCachePolicy.CACHE_FIRST)
-  .enqueue(new ApolloCall.Callback<FeedQuery.Data>() {
-
-    @Override public void onResponse(@NotNull Response<FeedQuery.Data> dataResponse) {
-      ...
-    }
-
-    @Override public void onFailure(@NotNull Throwable t) {
-      ...
-    }
-  }); 
-```
-
-**IMPORTANT:** Caching is provided only for `query` operations. It isn't available for `mutation` operations.
+## Downloading a schema.json file
 
-There are four available cache policies `HttpCachePolicy`:
+You can get a schema.json file by running an introspection query on your endpoint. Else, you can use the apollo CLI. See [here](https://github.com/apollographql/apollo-tooling#apollo-clientdownload-schema-output) for instructions.
 
-- `CACHE_ONLY` - Fetch a response from the cache only, ignoring the network. If the cached response doesn't exist or is expired, then return an error.
-- `NETWORK_ONLY` - Fetch a response from the network only, ignoring any cached responses.
-- `CACHE_FIRST` - Fetch a response from the cache first. If the response doesn't exist or is expired, then fetch a response from the network.
-- `NETWORK_FIRST` - Fetch a response from the network first. If the network fails and the cached response isn't expired, then return cached data instead.
+## Intellij Plugin
 
-For `CACHE_ONLY`, `CACHE_FIRST` and `NETWORK_FIRST` policies you can define the timeout after what cached response is treated as expired and will be evicted from the http cache, `expireAfter(expireTimeout, timeUnit)`.`
+The [JS Graphql Intellij Plugin](https://jimkyndemeyer.github.io/js-graphql-intellij-plugin/) provides auto-completion, error highlighting, and go-to-definition functionality for your graphql files. You can create a [.graphqlconfig](https://jimkyndemeyer.github.io/js-graphql-intellij-plugin/docs/developer-guide#working-with-graphql-endpoints-and-scratch-files) file in order to use GraphQL scratch files to work with your schema outside product code, e.g. by writing temporary queries to test resolvers.
```

Comment:
This is amazing information. We have been using this for a while. Thanks for adding it 🙌

Reply:
It was there before actually, just burried in the 800+ lines of the README.md